crypto/src/hash/mod.rs

```diff
@@ -23,12 +23,16 @@ pub use rescue::{Rp62_248, Rp64_256};
 ///
 /// This trait defined hash procedures for the following inputs:
 /// * A sequence of bytes.
+/// * Collision resistance of the hash function measured in bits.
```

Comment:
This bullet point feels out of place here (it does not describe a hashing procedure). I'll put it into a separate paragraph after the list. Or we can get rid of this altogether.

Also, there is a typo on line 24 (which was there before this PR): "defined" should be "defines".